test/runtests.jl

```diff
@@ -4,3 +4,8 @@ using Test
 @testset "MeasureTheory.jl" begin
     # Write your own tests here.
 end
+
+@testset "Kernel" begin
+    κ = MeasureTheory.kernel(x->(x,), MeasureTheory.Dirac)
+    @test rand(κ(1.1)) == 1.1
```

Comment:
Testing the special case of the deterministic kernel, just to see

Reply:
Great!!